<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionPage</class>
 <widget class="QWidget" name="SessionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1358</width>
    <height>930</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">#courtFrame {
	background: palette(light);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QPushButton" name="checkInButton">
        <property name="toolTip">
         <string>Check in the game</string>
        </property>
        <property name="text">
         <string>Check in</string>
        </property>
        <property name="icon">
         <iconset resource="../res/icons/icons.qrc">
          <normaloff>:/icons/System/login-box-line.svg</normaloff>:/icons/System/login-box-line.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,5,0,2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
       <item>
        <widget class="QLabel" name="timeLabel">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="wardenOptionButton">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../res/icons/icons.qrc">
           <normaloff>:/icons/System/more-fill.svg</normaloff>:/icons/System/more-fill.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>On the courts</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="courtFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Having a break</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="benchList">
       <property name="flow">
        <enum>QListView::TopToBottom</enum>
       </property>
       <property name="isWrapping" stdset="0">
        <bool>true</bool>
       </property>
       <property name="resizeMode">
        <enum>QListView::Adjust</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
